nautobot/extras/forms/mixins.py

```diff
@@ -137,6 +140,44 @@ def __init__(self, *args, **kwargs):
             self.custom_fields.append(field_name)
 
 
+class NoteFormBase:
+    """Base fore the NoteModelFormMixin and NoteModelBulkEditFormMixin."""
+
+    def append_field(self):
+        self.fields["object_note"] = CommentField()
+        self.fields["object_note"].label = "Note"
+
+    def save_note(self, *, instance, user):
+        if "object_note" in self.cleaned_data:
```

Comment:
Just checking - if no note text is specified (the field is left blank), we don't create an empty Note, do we?

Reply:
Good catch, added additional to the conditional to ensure this doesn't happen.